distribution/lib/Standard/Visualization/0.0.0-dev/src/Table/Visualization.enso

```diff
@@ -6,7 +6,7 @@ import Standard.Table.Data.Row.Row
 import Standard.Table.Data.Table.Table as Dataframe_Table
 
 import Standard.Database.Data.DB_Column.DB_Column
-import Standard.Database.Data.Table.Table as Database_Table
+import Standard.Database.Data.DB_Table.DB_Table as Database_Table
```

Comment:
```suggestion
import Standard.Database.Data.DB_Table.DB_Table
```

I think this alias is redundant and shall we removed.

Since there's no longer name clashing, there's no point in renaming `DB_Table` into `Database_Table` - having these 2 different names may actually be confusing now IMO.

Reply:
Next MR removes aliases.